gatherers/theme-color.js

```diff
@@ -26,7 +27,10 @@ class ThemeColor extends Gather {
     return driver.querySelector('head meta[name="theme-color"]')
       .then(node => node && node.getAttribute('content'))
       .then(themeColorMeta => {
-        return {themeColorMeta};
+        if (themeColorMeta && validColor(themeColorMeta)) {
```

Comment:
we should test this behavior :) (I know it is not complex, but think of the test as ensuring the contract is held up)